sqlmesh/core/test/definition.py

```diff
@@ -76,19 +83,21 @@ def setUp(self) -> None:
                     v = v.real if hasattr(v, "real") else v
                     columns_to_types[i] = parse_one(type(v).__name__, into=exp.DataType)
 
-            table = exp.to_table(table_name)
-            if table.db:
+            fqft = _fully_qualified_test_fixture_table(table_name)
```

Comment:
wtft is `fqft` lol?

Reply:
I was thinking you were gonna love this based on the fqt feedback 😛